test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPackCommandTest.cs

```diff
@@ -5984,6 +5984,55 @@ public void PackCommand_PackIcon_MissingIconFile_Fails()
             TestPackIconFailure(testDirBuilder, NuGetLogCode.NU5019.ToString());
         }
 
+        [Theory]
+        [InlineData(".jpg")]
+        [InlineData(".png")]
+        [InlineData(".PnG")]
```

Comment:
I appreciate how detailed your tests are, but keep in mind whether all these tests are still relevant. 
Ex. .jpeg, .JpEg, abc.JpEg.